homeassistant/components/steam_online/sensor.py

```diff
@@ -95,12 +117,27 @@ def _get_current_game(self):
         if game_extra_info:
             return game_extra_info
 
-        if game_id and game_id in self._steam_app_list:
-            # The app list always returns a tuple
-            # with the game id and the game name
-            return self._steam_app_list[game_id][1]
+        if not game_id:
+            return None
+
+        app_list = self.hass.data[APP_LIST_KEY]
+        try:
+            _, res = app_list[game_id]
+            return res
+        except KeyError:
+            pass
+
+        # Try reloading the app list, must be a new app
+        app_list = self._steamod.apps.app_list()
```

Comment:
If a user incorrectly configures a game, it will hit this each update of that account. As expected?

Reply:
The game ID is the current game that the user is playing, pulled from the app not from the config.